mvn-extension/src/test/java/org/arquillian/smart/testing/mvn/ext/dependencies/StrategyDependencyResolverTest.java

```diff
@@ -97,4 +102,65 @@ public void should_overwrite_default_properties_by_those_specified_in_the_file_a
                 "org.arquillian.smart.testing:smart-testing-strategy-affected:0.0.2-SNAPSHOT",
                 "org.arquillian.smart.testing:smart-testing-strategy-failed:0.0.13-SNAPSHOT");
     }
+
+    @Test
+    public void should_add_surefire_dependency()
```

Comment:
They are testing `DependencyResolver` so probably belong to the other test class. But many thanks for writing them!

Would it make sense to add one covering `-SNAPSHOT` dependency too?